dbt_platform_helper/commands/pipeline.py

```diff
@@ -190,10 +163,3 @@ def _create_file_from_template(
     ).render(template_data)
     message = mkfile(base_path, pipelines_dir / file_name, contents, overwrite=True)
     click.echo(message)
-
-
-def _safe_load_config(filename, schema):
-    try:
-        return load_and_validate_config(filename, schema)
-    except ParserError:
-        abort_with_error(f"The {filename} file is invalid")
```

Comment:
Moved this exception handling into the validation.py function ' load_and_validate_platform_config' so it can be used by anything that wants to validate platform-config.yml